torchaudio/datasets/vctk.py

```diff
@@ -53,15 +53,14 @@ class VCTK(Dataset):
     """
     Create a Dataset for VCTK. Each item is a tuple of the form:
     (waveform, sample_rate, utterance, speaker_id, utterance_id)
-
     Folder `p315` will be ignored due to the non-existent corresponding text files.
     For more information about the dataset visit: https://datashare.is.ed.ac.uk/handle/10283/3443
     """
 
     _folder_txt = "txt"
-    _folder_audio = "wav48"
+    _folder_audio = "wav48_silence_trimmed"
     _ext_txt = ".txt"
-    _ext_audio = ".wav"
+    _ext_audio = ".flac"
```

Comment:
Assuming that this is correct. I wonder how we should approach backward compatibility.

If we just update this, people who have already downloaded the previous format of the dataset would face error. Ideally we do not want to disturb and force those people to re-download it. Do you have a thought?

Reply:
Added backward compatibility for older versions of datasets. Although it's not the best and clean implementation, it will work in 99.9% scenarios.